rafs/src/metadata/md_v6.rs

```diff
@@ -68,6 +68,52 @@ impl RafsSuper {
         }
     }
 
+    pub(crate) fn is_inlay_prefetch_all(&self, r: &mut RafsIoReader) -> RafsResult<bool> {
+        if self.meta.is_v6() {
+            let hint_entries = self.meta.prefetch_table_entries as usize;
+            if hint_entries != 1 {
+                return Ok(false);
+            }
+
+            let mut prefetch_table = RafsV6PrefetchTable::new();
+            prefetch_table
+                .load_prefetch_table_from(r, self.meta.prefetch_table_offset, hint_entries)
+                .map_err(|e| {
+                    RafsError::Prefetch(format!(
+                        "Failed in loading hint prefetch table at offset {}. {:?}",
+                        self.meta.prefetch_table_offset, e
+                    ))
+                })?;
+
+            if prefetch_table.inodes[0] as u64 == self.superblock.root_ino() {
+                Ok(true)
+            } else {
+                Ok(false)
+            }
+        } else {
```

Comment:
`md_v6.rs` should only contains code for v6, so is this branch redundant?

Reply:
Seems like it's used for both v5 and v6, so it would be better to move into a common source file.

Reply:
Yes. Could we do the code movement on the master branch as I am planning to commit these changes to the master in the future?